resources/user-agents/browsers/firefox/firefox-3-1.json

```diff
@@ -23,7 +23,8 @@
         "Tables": "true",
         "Cookies": "true",
         "JavaScript": "true",
-        "CssVersion": "3"
+        "CssVersion": "3",
+        "Beta": "true"
```

Comment:
3.1 never had an official release (3.5 was the next major release), so I marked this as beta.

https://en.wikipedia.org/wiki/Firefox_3.5#Beta_and_release